spec/factories/groups.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+FactoryGirl.define do
+  factory :group do
+    size 1
+    status 'full'
+    association :leader, factory: :student_in_draw
+    draw { leader.draw }
```

Comment:
Do we need to add `members { [leader] }` as well, or do you want to do that when we add the validations / callbacks that ensure that the leader is also a member?